instance_config.yaml

```diff
@@ -13,11 +13,11 @@
 DATABASES:
     default:
         ENGINE: 'django.db.backends.mysql'
```

Comment:
Do we still need to set up an sql db named 'pipeline' manually while setting up the local env?